tests/testsuite/publish.rs

```diff
@@ -2309,7 +2315,10 @@ fn api_error_200() {
 [PACKAGING] foo v0.0.1 ([ROOT]/foo)
 [PACKAGED] 4 files, [FILE_SIZE]B ([FILE_SIZE]B compressed)
 [UPLOADING] foo v0.0.1 ([ROOT]/foo)
-[ERROR] failed to publish to registry at http://127.0.0.1:[..]/
+[ERROR] failed to publish `foo` v0.0.1
+
+Caused by:
+  failed to publish to registry at http://127.0.0.1:37651/
```

Comment:
Note that the original error message had the port number redacted with `[..]`.  This avoids per-run information being embedded in the test output